x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/edit/index.tsx

```diff
@@ -332,75 +333,79 @@ const EditRulePageComponent: FC = () => {
 
   return (
     <>
-      <WrapperPage restrictWidth>
-        <DetectionEngineHeaderPage
-          backOptions={{
-            href: getRuleDetailsUrl(ruleId ?? ''),
-            text: `${i18n.BACK_TO} ${rule?.name ?? ''}`,
-            pageId: SecurityPageName.detections,
-          }}
-          isLoading={isLoading}
-          title={i18n.PAGE_TITLE}
-        />
-        {invalidSteps.length > 0 && (
-          <EuiCallOut title={i18n.SORRY_ERRORS} color="danger" iconType="alert">
-            <FormattedMessage
-              id="xpack.securitySolution.detectionEngine.rule.editRule.errorMsgDescription"
-              defaultMessage="You have an invalid input in {countError, plural, one {this tab} other {these tabs}}: {tabHasError}"
-              values={{
-                countError: invalidSteps.length,
-                tabHasError: invalidSteps
-                  .map((t) => {
-                    if (t === RuleStep.aboutRule) {
-                      return ruleI18n.ABOUT;
-                    } else if (t === RuleStep.defineRule) {
-                      return ruleI18n.DEFINITION;
-                    } else if (t === RuleStep.scheduleRule) {
-                      return ruleI18n.SCHEDULE;
-                    } else if (t === RuleStep.ruleActions) {
-                      return ruleI18n.RULE_ACTIONS;
-                    }
-                    return t;
-                  })
-                  .join(', '),
+      <WrapperPage>
+        <EuiFlexGroup direction="row" justifyContent="spaceAround">
+          <MaxWidthEuiFlexItem>
```

Comment:
NOTE: I did _not_ change a bunch of code here.

Instead I just changed:

```ts
<WrapperPage restrictWidth>
        <EuiFlexGroup direction="row" justifyContent="spaceAround">
          <MaxWidthEuiFlexItem>
```

to be:

```ts
<WrapperPage>
        <EuiFlexGroup direction="row" justifyContent="spaceAround">
          <MaxWidthEuiFlexItem>
```